app/containers/ArticleApp/ArticleApp.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import React from 'react';
+import Relay from 'react-relay';
+import ArticleList from './components/ArticleList';
+
+type Props = {
+  user: {
+    id: any,
+    articles: Array,
+  },
+};
```

Comment:
Why are you doing this? Does it help with Flow or is this standard somewhere?

Reply:
this allows the IDE to know what types `this.props.user.{id,articles}` are. Or do you mean why I pass those in as properties in the first place? I picked that up from the starter repo - I am not sure why they did that.